client/js/screens/Card/Card.js

```diff
@@ -1,22 +1,72 @@
-import React from 'react';
-import { View, Text, ImageBackground } from 'react-native';
+import React, { Component } from 'react';
+import { View, ImageBackground } from 'react-native';
+import { TabView, SceneMap, TabBar } from 'react-native-tab-view';
 import PropTypes from 'prop-types';
-import Barcode from 'react-native-barcode-builder';
+
+import CustomText from '../../components/CustomText';
+import CardTab from '../../components/CardTab';
+import HistoryTab from '../../components/HistoryTab';
+import RewardsTab from '../../components/RewardsTab';
+import { dimensions } from '../../config/styles';
 import styles from './styles';
 
-const Card = props => {
-  return (
-    <View style={styles.container}>
-      <ImageBackground
-        source={require('../../assets/images/logo_bg.png')}
-        style={styles.imgBg}
-      >
-        <Text>This is Card.</Text>
-        <Barcode value="Test Card" format="CODE128" />
-      </ImageBackground>
-    </View>
-  );
-};
+class Card extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      index: 0,
+      routes: [
+        { key: 'first', title: 'Card', accessibilityLabel: 'Card Tab' },
+        { key: 'second', title: 'History', accessibilityLabel: 'History Tab' },
+        { key: 'third', title: 'Rewards', accessibilityLabel: 'Rewards Tab' }
+      ]
+    };
+  }
+
+  _renderLabel(props) {
```

Comment:
if you are following this convention od putting _ in front of your methods then all of your methods should follow this convention (groupwide).

Reply:
Thanks for the tip and I have removed the underscore.